INN/umbrella-mstoday
# Add subscribe button to global and sticky nav

## Changes

This pull request makes the following changes:

- Adds a subscribe button to the global and sticky navs using the same approach we used for MWCIR https://github.com/INN/umbrella-mwcir/pull/34

Desktop:
![Screen Shot 2020-03-03 at 1 43 14 PM](https://user-images.githubusercontent.com/18353636/75808952-e8596f00-5d55-11ea-9735-72cf0727f38c.png)

Sticky:
![Screen Shot 2020-03-03 at 1 50 03 PM](https://user-images.githubusercontent.com/18353636/75808943-e5f71500-5d55-11ea-8982-288d6ed04592.png)

Mobile:
![Screen Shot 2020-03-03 at 1 43 26 PM](https://user-images.githubusercontent.com/18353636/75808950-e7c0d880-5d55-11ea-99f8-ae97d75f46fd.png)

## Why



For #66

## Testing/Questions

Features that this PR affects:

- Global and sticky nav


Questions that need to be answered before merging:

- [ ] Is `/subscribe/` the correct page to point the button to?

Steps to test this PR:

1. View the global and sticky navs on desktop and mobile and make sure it looks ok
